pkg/data/management/kev2_operator_data.go

```diff
@@ -0,0 +1,104 @@
+package management
+
+import (
+	"encoding/json"
+	"fmt"
+	"reflect"
+
+	"github.com/rancher/rancher/pkg/settings"
+	"github.com/rancher/rancher/pkg/types/config"
+	"k8s.io/apimachinery/pkg/api/errors"
+)
+
+type KEv2OperatorInfo struct {
+	Name          string `json:"name"`
+	OldDriverName string `json:"oldDriverName"`
+	Active        bool   `json:"active"`
+}
+
+var defaultKEv2Operators = []KEv2OperatorInfo{
+	{Name: "aks", OldDriverName: "azureKubernetesService", Active: true},
+	{Name: "eks", OldDriverName: "amazonElasticContainerService", Active: true},
+	{Name: "gke", OldDriverName: "googleElasticContainerService", Active: true},
+	{Name: "alibabacloud", OldDriverName: "", Active: false},
+}
+
+func syncOperatorDriverActiveState(management *config.ManagementContext) error {
```

Comment:
So part of the reason why I feel like settings are not the correct use case for this:
1. This need a relatedresource watch for kontainerdrivers, because right now if you disable one of the operators the setting doesn't get updated.
2. There doesn't appear to be any validation for the setting either, so nothing stops someone from potentially editing the setting to something the UI can't interpret.